test/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -131,10 +132,14 @@ public void ItPublishesFrameworkDependentWithRid(string args)
 
             var outputProgram = Path.Combine(outputDirectory.FullName, $"{testAppName}{Constants.ExeSuffix}");
 
-            new DotnetCommand()
-                .ExecuteWithCapturedOutput(Path.Combine(outputDirectory.FullName, $"{testAppName}.dll"))
-                .Should().Pass()
-                     .And.HaveStdOutContaining("Hello World");
+            var command = new TestCommand(outputProgram);
```

Comment:
Should we use `dotnet <app.dll>` here or really switch to the `app.exe`? Or both? Or should we have two separate tests for it?

Reply:
The framework-dependent without RID test case covers running it via the muxer since there isn't an apphost; I figured we wouldn't need to cover both scenarios from this test as it's more important to ensure the apphost activates correctly for fx-dependent.

Reply:
Cool. Sounds good.